sui_types/src/committee.rs

```diff
@@ -68,4 +70,34 @@ impl Committee {
         // then (N + 2) / 3 = f + 1 + k/3 = f + 1
         (self.total_votes + 2) / 3
     }
+
+    /// Given a sequence of (AuthorityName, value) for values that are ordered, provide the
+    /// value at the particular threshold by stake. This orders all provided values and pick
+    /// the appropriate value that has under it threshold stake. You may use the function
```

Comment:
Probably needs more explanation on the semantics of `robust_value`

Reply:
Added quite a bit more